WNPRC_EHR/test/src/org/labkey/test/tests/wnprc_ehr/WNPRC_EHRTest.java

```diff
@@ -1682,13 +1682,14 @@ public void testMprDataEntry()
         waitForElement(Locator.tagWithText("span", "Treatments & Procedures"), WAIT_FOR_JAVASCRIPT);
         waitForElement(Locator.name("Id"), WAIT_FOR_JAVASCRIPT);
         waitForElement(Locator.name("title"), WAIT_FOR_JAVASCRIPT);
+        _helper.setDataEntryField("title", MPR_TASK_TITLE);
+        waitAndClick(Locator.name("title"));
         _extHelper.selectComboBoxItem("Assigned To:", BASIC_SUBMITTER.getGroup() + "\u00A0"); // appended with a nbsp (Alt+0160)
         _extHelper.setExtFormElementByLabel("Id:", PROJECT_MEMBER_ID + "\t");
         click(Locator.xpath("//div[./label[normalize-space()='Id:']]//input"));
         waitForElement(Locator.linkWithText(PROJECT_MEMBER_ID), WAIT_FOR_JAVASCRIPT);
-        _helper.setDataEntryField("title", MPR_TASK_TITLE);
-        waitAndClick(Locator.name("title"));
-
+        _helper.setDataEntryField("performedby", FULL_SUBMITTER.getEmail());
+        getArtifactCollector().dumpPageSnapshot("Edited_title_for_MPR task");
```

Comment:
You can safely delete this screenshot now if you like.

Reply:
Yup, was planning to after one successful run